config/samples/kamaji_v1alpha1_tenantcontrolplane.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     tenant.clastix.io: k8s-126
 spec:
+  dataStore: nats-notls
```

Comment:
This must be removed, by default Kamaji install just the `default` one, this hard-coded value can create issues for new-comers.